MediaAdminBundle/Tests/FileUtils/AbstractFileUtilsManager.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace OpenOrchestra\MediaAdminBundle\Tests\FileUtils;
+
+use Phake;
+use Imagick;
+
+/**
+ * Class AbstractFileUtilsManager
+ */
+abstract class AbstractFileUtilsManager extends \PHPUnit_Framework_TestCase
+{
+    protected $fixturesPath;
+
+    protected $generatedFiles = array();
+
+    /**
+     * Set up the test
+     */
+    public function setUp()
+    {
+        $this->fixturesPath = __DIR__ . '/../Fixtures/';
+    }
+
+    /**
+     * Check if a file has been correctly generated and tag it to be deleted on tearDown
+     * 
+     * @param string $expectedFile
+     * @param string $generatedFile
+     */
+    protected function assertFileCorrectlyGenerated($expectedFile, $generatedFile)
+    {
+        $this->generatedFiles[] = $generatedFile;
+
+        $this->assertFileExists($generatedFile);
+//        $this->assertFileEquals($expectedFile, $generatedFile);
```

Comment:
I remember this issue :).

I have no better way to compare two images at the time..